### DISCUSSION TITLE
Reapplying for student status - "process my application" button just clears the form and doesn't submit anything

### DISCUSSION THREAD

**Discussion by EspenFredrick** [community/community#129004] [GitHub Education]
### Hi everyone,

I am away from campus visiting my parents for the summer because I don't start teaching summer labs until next month, and of course GitHub is throwing a fit about that. I want to include a photo of an email saying labs start later this summer, so I need to take a photo of my laptop screen.

I've tried the most up-to-date Safari and Chrome on iOS 17.5.1 on two different networks AND on mobile data and every time I submit the photo the form just refreshes. No confirmation that I submitted, no automatic email reply, no accept/denied letter, nothing. My next step is to try another device bit I simply don't have one. Anyone else having this issue?

**Comment by davevad93** [comment]
Hi @EspenFredrick ,

Take a look at this discussion #126276 , this user solved it by taking a photo from his phone.
